dipy/tracking/local/localtracking.py

```diff
@@ -111,12 +111,9 @@ def _tracker(self, seed, first_step, streamline):
     def __iter__(self):
         # Make tracks, move them to point space and return
         track = self._generate_streamlines()
-        seeds = None
-        if self.save_seeds:
-            track, seeds = zip(*track)
-        return utils.move_streamlines(track,
-                                      self.affine,
-                                      seeds=seeds)
+
+        return utils.transform_tracking_output(track, self.affine,
```

Comment:
What is this function? Why not `transform_streamlines` from `dipy.tracking.streamlines`?

Reply:
My reasoning was that transform_streamlines should do one thing and one thing only, to stay as simple as possible.

This function takes a generator as a input that can be a tuple or not, and in 99% of the case where people want to transform_streamlines it is just that.
This function with the generator, the tuple and the saving_seeds option is used only once in all dipy and is specifically for the tracking.

I think merging the two function would add to the confusion. And complexify a function that should remain as simple as simple (transform_streamlines)

Reply:
ok, make sense!